internal/gengapic/markdown_test.go

```diff
@@ -74,8 +74,8 @@ func TestMDPlain(t *testing.T) {
 			want: "html with a softbreak (at /link/to/some#thing) \n test",
 		},
 		{
-			in:   "link to [a search engine](https://www.google.com) with request type [Search][foo.bar.v1.Search].",
-			want: "link to a search engine (at https://www.google.com) with request type Search.",
+			in:   "link to [a search engine](https://www.google.com) with request type [Search][foo.bar_baz.v1.Search], [biz][].",
```

Comment:
Would you mind adding another test case for something that was matched before this change, but isn't matched now?

Reply:
Done! I expanded it some more. But basically, if there were multiple references in a block, the previous regex would match the link text to keep as everything between the first square bracket and the closing bracket of the last set, keeping essentially everything.